src/libs/Network.js

```diff
@@ -56,13 +57,16 @@ function processPersistedRequestsQueue() {
     const tasks = _.map(persistedRequests, request => processRequest(request)
         .then((response) => {
             if (response.jsonCode !== CONST.HTTP_STATUS_CODE.SUCCESS) {
-                throw new Error('Persisted request failed');
+                throw new Error(`Persisted request failed due to jsonCode: ${response.jsonCode}`);
             }
 
             NetworkRequestQueue.removeRetryableRequest(request);
         })
-        .catch(() => {
+        .catch((error) => {
+            const Log = getLogger();
```

Comment:
Maybe just move this to the very top of the file so that it's only called once?

Reply:
It wouldn't work 
![image](https://user-images.githubusercontent.com/12156624/153446601-39d66d66-5abb-4fd9-bc29-61706f30ac78.png)

Another script (API) first reads (imports) the `Network.js` script and evaluates all to level expression e.g. `const Log = getLogger()` 
And only then assigns value to be returned by `getLogger()`

Reply:
Ah, OK. Makes sense

Reply:
Removed `const Log = getLogger()`
Replaced usages to be like `getLogger().info(...)` 

Is that better?